sklearn/cluster/tests/test_optics.py

```diff
@@ -459,3 +390,12 @@ def test_precomputed_dists():
 
     assert_allclose(clust1.reachability_, clust2.reachability_)
     assert_array_equal(clust1.labels_, clust2.labels_)
+
+
+def test_processing_order():
+    """Unit test for bug in ca. version 0.20, see #12090"""
```

Comment:
I guess we can remove this comment. This is not a bug because OPTICS is not released.

Reply:
I'd keep the issue number in there, in case someone wonders where this test case comes from.
How about naming this test_processing_order_issue12090 and dropping the comment?

Reply:
Maybe remove ``bug`` and ``version 0.20`` ? It's not a bug in 0.20. Another thing is that this one is actually a different bug, so I doubt whether it's good to link to #12090.